cmake/kokkos_functions.cmake

```diff
@@ -60,6 +60,444 @@ FUNCTION(kokkos_option CAMEL_SUFFIX DEFAULT TYPE DOCSTRING)
 
 ENDFUNCTION()
 
+FUNCTION(kokkos_append_config_line LINE)
+  GLOBAL_APPEND(KOKKOS_TPL_EXPORTS "${LINE}")
+ENDFUNCTION()
+
+MACRO(kokkos_export_cmake_tpl NAME)
+  #CMake TPLs are located with a call to find_package
+  #find_package locates XConfig.cmake files through
+  #X_DIR or X_ROOT variables set prior to calling find_package
+
+  #If Kokkos was configured to find HPX through a _DIR variable
```

Comment:
Why exactly do we talk about HPX here? I see that this is the only TPL `kokkos_export_cmake_tpl` is currently called for, but did you mean to use ${NAME} instead?

Reply:
Yeah, my brain wanted to say TPL, but was disobeyed and I typed HPX. Thanks for catching it. Fixed.